### PR TITLE
FEM: feminout: XDMF and XML writing: fixed minor errors and format now in line with meshio

### PR DESCRIPTION
**Pull request by joha2**
* XDMF: capital "V" in version attribute in XDMF tag
* XDMF: capitalizing element names to be in line with specification
* XML: removed data section to prevent warning in meshio reading

Thank you for creating a pull request to contribute to FreeCAD! Place an "X" in between the brackets below to "check off" to confirm that you have satisfied the requirement, or ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10) if there is something you don't understand.

- [X]  Your Pull Request meets the requirements outlined in section 5 of [CONTRIBUTING.md](https://github.com/FreeCAD/FreeCAD/blob/master/CONTRIBUTING.md) for a Valid PR

Please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [1.0 Changelog Forum Thread](https://forum.freecad.org/viewtopic.php?f=10&t=69438).

---

### CODE REVIEW

File: src/Mod/Fem/feminout/readFenicsXML.py [hunk 1]
```diff
@@ -244,7 +244,7 @@ def invertdict(dic):
         for (elm, numbers) in list(element_dict.items()):
             lst = sorted(list(numbers.items()), key=lambda x: x[0])
             if lst != []:
-                Console.PrintWarning(elm, " min: ", lst[0], " max: ", lst[-1], "\n")
+                Console.PrintWarning("{} min: {} max: {}\n".format(elm, lst[0], lst[-1]))
```
Comment (benj5378) [inline src/Mod/Fem/feminout/readFenicsXML.py:247]:
```suggestion
                Console.PrintWarning(f"{elm} min: {lst[0]} max: {lst[-1]}\n")
```
Make it even better!

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9240">pipeline status</a> for feature branch [PR_9240](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9240). Pipeline [838250107](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/838250107) was triggered at [b41bc9d](https://github.com/FreeCAD/FreeCAD/pull/9240/commits/b41bc9defe199e602b4130106e0d6a0c973f5195). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9240">pipeline status</a> for feature branch [PR_9240](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_9240). Pipeline [838280771](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/838280771) was triggered at [0fd08f9](https://github.com/FreeCAD/FreeCAD/pull/9240/commits/0fd08f9baad46689062ba4970cf360311122458c). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Fem/feminout/readFenicsXML.py (hunk 1 above)
Comment (joha2) [reply src/Mod/Fem/feminout/readFenicsXML.py]: @luzpaz Thanks for the heads up. It's a little late, but I will mark it as solved :-)

**Pull request merged**